tests/unit/tests_high_five_base.cpp

```diff
@@ -426,6 +426,56 @@ TEST_CASE("Test groups and datasets") {
     }
 }
 
+#if H5_VERSION_GE(1, 10, 1)
+TEST_CASE("FileSizeAndUnusedSpace") {
```

Comment:
Please split the test in two. One for the untracked file and one for the tracked variant.